src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -99,6 +99,14 @@ namespace UtilityRoutines {
     bool outputErrorHeader(true);
     ObjexxFCL::gio::Fmt fmtLD("*");
     ObjexxFCL::gio::Fmt fmtA("(A)");
+    std::string appendPerfLog_headerRow("");
```

Comment:
I see that these are only in the cc file, not the header.  That is a very good thing.  There is no reason the implementation of this needs to be exposed to any other file, and the clear_state function clears these.  :+1: